docs/content/Schema/Dynamic-Schema-Creation.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Dynamic Schema Creation
+permalink: /schema/dynamic-schema-creation
+category: Data Schema
+menuOrder: 2
+---
+
+Cube.js allows schemas to be created on-the-fly using a special
+[`asyncModule()`][ref-async-module] function only available in the [schema
```

Comment:
It's really a very small detail but I'd like to ask whether we should add parentheses in the end of function names. I understand why we can do that but the drawback is that one might think that they are argument-less functions which is rarely true. WDYT, @hassankhan? (Anyway, regardless of the side you take :), it's worth documenting how we spell function names.)

Reply:
@igorlukanin Great point, I've add parentheses mainly to distinguish functions from variables/properties. I usually try to link out to the function's documentation too where possible. Happy to go either way though 😀

Reply:
I believe it is a quite common notation for functions to display name with parentheses.